frontend/src/index.js

```diff
@@ -1,10 +1,6 @@
-/**
- * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
- *
- * This source code is licensed under the Apache 2.0 license found in the
- * LICENSE file in the root directory of this source tree.
-*/
-
+import { shopNumber } from '@shopgate/pwa-common/helpers/config';
+import { getUserData } from '@shopgate/pwa-common/selectors/user';
+import { userDataReceived$ } from '@shopgate/pwa-common/streams/user';
```

Comment:
The `pwa-common` module isn't available within the `package.json`

Reply:
done